src/machine/board_trinket.go

```diff
@@ -51,9 +54,8 @@ var (
 	}
 )
 
-//go:export SERCOM1_IRQHandler
-func handleUART1() {
-	defaultUART1Handler()
+func init() {
+	UART1.interrupt = interrupt.New(sam.IRQ_SERCOM0, UART1.handleInterrupt)
```

Comment:
I found a bug here: UART1 is defined with SERCOM0 but the interrupt fired on SERCOM1. This PR fixes that.